backend/test/integration/openai.test.ts

```diff
@@ -366,134 +375,4 @@ describe('OpenAI Integration Tests', () => {
 			mockCreateChatCompletion.mockRestore();
 		}
 	);
-
-	test('GIVEN the output filtering defence is active WHEN the bot responds with a message containing a phrase in the list THEN the defence is triggered and the message is blocked', async () => {
```

Comment:
Remind me, why did you remove these tests? Are they re-added in a different PR?

Reply:
Yes I had misunderstood and thought George and I would merge branches. Waiting on George confirming after resolving conflicts before adding back in/or not

Reply:
the tests are being added back in in George's branch [here](https://github.com/ScottLogic/prompt-injection/pull/740/files#r1452379574)